react/Footer/Footer.less

```diff
@@ -1,24 +1,19 @@
 @import (reference) "~seek-style-guide/theme";
 @import (reference) "./Footer.common.less";
 
-@classic-ia: ~"only screen and (min-width: 1025px)";
-
 .root {
   background-color: @sk-footer;
   width: 100%;
 }
 
 .content {
-  padding: (@grid-row-height * 4) @grid-gutter-width;
+  padding: (@grid-row-height * 4) (@grid-gutter-width / 2);
```

Comment:
10px padding is what is currently used on the header to keep the logo/text of the edge of the screen. This is case particular along with matching that 20px is too much as it looks like just another normal PageBlock.